node/pkg/watchers/evm/verify_chain_config/verify.go

```diff
@@ -77,25 +82,40 @@ func verifyForEnv(env common.Environment, chainID vaa.ChainID) {
 			if entry.Entry.PublicRPC == "" {
 				fmt.Printf("Skipping %v %v because the rpc is null\n", env, entry.ChainID)
 			} else {
+				fmt.Printf("Verifying %v %v...\n", env, entry.ChainID)
+
+				fmt.Printf("   Verifying EVM chain ID for %v %v ", env, entry.ChainID)
 				evmChainID, err := evm.QueryEvmChainID(ctx, entry.Entry.PublicRPC)
 				if err != nil {
-					fmt.Printf("ERROR: Failed to query EVM chain ID for %v %v: %v\n", env, entry.ChainID, err)
+					fmt.Printf("\u2717\n   ERROR: Failed to query EVM chain ID for %v %v: %v\n", env, entry.ChainID, err)
```

Comment:
You don't want the 'x' on the same line?

Reply:
I think this is right. I want the "x" at the end of the previous line and the error message on a new line.